vaccine_feed_ingest/runners/_shared/parse.py

```diff
@@ -60,7 +62,21 @@ def _output_ndjson(json_list: List[dict], out_filepath: pathlib.Path) -> None:
             fout.write("\n")
 
 
+def _prepmod_find_data_item(parent, label, offset):
+    try:
+        row_matches = [
+            x for x in parent.find_all(["p", "div"]) if label in x.get_text()
+        ]
+        content = row_matches[-1].contents[offset]
+        return (
+            content.strip() if isinstance(content, str) else content.get_text().strip()
+        )
+    except Exception:
+        return ""
```

Comment:
Is it it worth logging this case, or is it expect error that doesn't need to be logged?

Reply:
Oh, I see! This is erroring if `row_matches` is an empty list, or contents is empty.

I am going to approve and merge, but if you have a chance to get back to this an improvement would be to change this to test for that case directly

e.g.
```
if row_matches and row_matches[-1].contents:
    content = row_matches[-1].contents[offset]
else:
    return ""
```